asf_heat_pump_affordability/notebooks/gshp_analysis.py

```diff
@@ -0,0 +1,447 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: asf_heat_pump_affordability
+#     language: python
+#     name: python3
+# ---
+
```

Comment:
Maybe add some context at the top of this notebook mentioning what is the purpose of this analysis, for future reference. You could even link the doc you created for Max - I'm guessing it will only be accessible to those at Nesta anyway. For me, reading through your notes really helped.